coderd/notifications/notifier.go

```diff
@@ -22,6 +24,10 @@ import (
 	"github.com/coder/coder/v2/coderd/database"
 )
 
+const (
+	NotificationsDefaultLogoURL = "https://coder.com/coder-logo-horizontal.png"
```

Comment:
Out of scope for this PR, but wouldn't it be better to link to a resource on the deployment here? (`https://my-deployment/logo.png`)

We can't guarantee this URL will keep working, on the deployment that would at least mean the deployment is down anyway or unreachable.

Reply:
Good idea 👍